pkg/console/subresource/configmap/brand_custom_logo.go

```diff
@@ -0,0 +1,40 @@
+package configmap
+
+import v1 "github.com/openshift/api/operator/v1"
+
+// borrowed from the image package
+// image.RegisterFormat()
+var commonImageHeaders = []string{
```

Comment:
are we going to use these in `IsLikelyCommonImageFormat()` ?

Reply:
Yup.  Is a placeholder for now, but thats the idea. I did a bit of research into this but didn't quite have time to implement.  Is a nice-to-have bit of extra func I'll prob add during bug fix time.